### PR TITLE
Fixed scope as null on IssetOnPropertyObjectToPropertyExistsRector.

### PR DESCRIPTION
**Pull request by eusonlito**
Related with issue #5090

### CODE REVIEW

File: rules/code-quality/src/Rector/Isset_/IssetOnPropertyObjectToPropertyExistsRector.php [hunk 1]
```diff
@@ -87,8 +87,13 @@ public function refactor(Node $node): ?Node
             /** @var Expr $object */
             $object = $issetVar->var->getAttribute(AttributeKey::ORIGINAL_NODE);
 
-            /** @var Scope $scope */
+            /** @var Scope $scope|null */
```
Comment (samsonasik) [inline rules/code-quality/src/Rector/Isset_/IssetOnPropertyObjectToPropertyExistsRector.php:90]:
```suggestion
            /** @var Scope|null $scope */
```

File: rules/code-quality/src/Rector/Isset_/IssetOnPropertyObjectToPropertyExistsRector.php [hunk 2]
```diff
@@ -87,8 +87,13 @@ public function refactor(Node $node): ?Node
...
             $scope = $object->getAttribute(AttributeKey::SCOPE);
+
+            if (! $scope) {
```
Comment (samsonasik) [inline rules/code-quality/src/Rector/Isset_/IssetOnPropertyObjectToPropertyExistsRector.php:93]:
```suggestion
            if ($scope === null) {
```

**Comment by samsonasik** [comment]
@eusonlito I cherry-picked your commit at PR #5094

**Comment by eusonlito** [comment]
Thanks!

**Pull request closed without merging**